docs/user/openstack/README.md

```diff
@@ -646,6 +646,24 @@ If you ran the installer with a [custom CA certificate](#self-signed-openstack-c
 oc edit configmap -n openshift-config cloud-provider-config
 ```
 
+### Enabling Octavia for Load Balancer Services
+
+OpenShift supports creating load balancer service types, and ingress controllers with load balancers as a backend using octavia as a day 2 operation. However, there are a few known issues to be aware of: only HTTP, HTTPS, and TCP traffic is supported, and the floating IPs attached to your Octavia load balancers will not be deleted if you delete your cluster.
+
+For non-kuryr deployments, you will have to enable Octavia. To do this, edit the cloud provider config:
```

Comment:
This is misleading, it may imply that this setting will enable Octavia on your OpenStack cloud. I'd change it to "enable Octavia integration" Also the disclaimer about Kuryr should got somewhere in the beginning of whole section. We just got to say that enabling Octavia integration only makes sense for non-Kuryr deployments.